pwa/src/templates/categoryDetailTemplate/CategoryDetailTemplate.tsx

```diff
@@ -14,6 +13,7 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faGripVertical, faLayerGroup, faTable, faTags } from "@fortawesome/free-solid-svg-icons";
 import { FiltersContext } from "../../context/filters";
 import { ComponentResultTemplate } from "../templateParts/resultsTemplates/ComponentResultsTemplate";
+import { Button, Document, Heading1, Heading3, Paragraph, Link } from "@utrecht/component-library-react";
```

Comment:
Since you have changed to import of the `Button` component the icons do have to correct spacing anymore.
The icon of the button should have a gap between the text and the icon itself.
To fix this you can remove the ` > span` in the css.

![image](https://user-images.githubusercontent.com/43807324/225370822-38f05bdb-da04-447f-827d-2e9c22b3c5b4.png)

Reply:
The button also does not have any way of showing if its active or not like we have on the homepage 
![image](https://user-images.githubusercontent.com/43807324/225370064-ccf8f2d9-c5f2-42f3-aa3d-780411329d65.png)
if the button is active its green and if its not active the colors are inverted.